### ISSUE TITLE
mysql memory consumption

### ISSUE THREAD

**Issue by guypaskar** [mysqljs/mysql#842]
Can you please help me and tell how much memory an sql row should take on node?

I have a large data set that needs to be processed on node. I perform several individuals queries and process them directly at each call to my app.

say each query returns 1,000,000 rows that weight about 80mb inside mysql. I save the data (the raw data on node) and I see that memory is increasing to large numbers (more than I would have expected)

So basically I have two questions:
1) could it be that there is a memory leak due to the fact that I always hold a reference to the raw data?
2)if a row size is 5kb on mysql what would be it size on node as an object?

Thanks,
Guy
